app/controllers/shopify_app/sessions_controller.rb

```diff
@@ -38,12 +40,15 @@ def authenticate
       copy_return_to_param_to_session
 
       if embedded_redirect_url?
+        ShopifyApp::Logger.debug("Embedded URL within / authenticate")
         if embedded_param?
+          ShopifyApp::Logger.debug("Embedded param")
```

Comment:
I think we can just remove this since we already have a log in `redirect_for_embedded`. 

I think the original idea was to separate out all of the conditionals in the `authenticate` method.